showcase.yml

```
@@ -244,9 +244,16 @@ sites:
     url: https://en.neumann.tokyo
     description: The corporate website of GK Neumann in Japan
     code: https://github.com/neumann-tokyo/en-neuman-tokyo-website
-  
+
   - title: minimalist blog template
     img: /img/showcase/minimalist-blog-image.webp
     url: https://minimalist-blog-x8c5ngg3kzt0.deno.dev/
     description: Minimalist blog template built for lume 
-    code: https://github.com/officialrajdeepsingh/Minimalist-blog
\ No newline at end of file
+    code: https://github.com/officialrajdeepsingh/Minimalist-blog
+
+  - title: Lume Deno Template
+    img: /img/showcase/LumeTemplate.webp
+    url: https://lume-deno-template.deno.dev/
+    description: eAnother Lume Template/ With CSS & Blog (MD) Support
```

I think there's a typo here:
```suggestion
    description: Another Lume Template/ With CSS & Blog (MD) Support
```